File: _프로그래머스/01-20/18. 문자열 내림차순으로 배치하기.py
"""
* Title: 문자열 내림차순으로 배치하기
https://school.programmers.co.kr/learn/courses/30/lessons/12917
"""


def solution(s):
    ls = list(str(s))
    ls.sort(reverse=True)
    answer = ''.join(ls)

    return answer

File: _프로그래머스 js/18. 문자열 내림차순으로 배치하기.js
/*
* Title: 문자열 내림차순으로 배치하기
https://school.programmers.co.kr/learn/courses/30/lessons/12917
*/
const solution = (n) => {
  const strls = Array.from(n);
  strls.sort().reverse();

  return strls.join('');
};

// "gfedcbZ"
console.log(solution('Zbcdefg'));
